AssemblerLib/LocalToGlobalIndexMap.h

```diff
@@ -65,6 +82,16 @@ class LocalToGlobalIndexMap
     LineIndex columnIndices(std::size_t const mesh_item_id) const;
 
 private:
+
+    /// Private constructor used by internally created local-to-global index
+    /// maps. The mesh_component_map is passed as argument instead of being
+    /// created by the constructor.
+    explicit LocalToGlobalIndexMap(
+        std::vector<MeshLib::MeshSubsets*> const& mesh_subsets,
+        std::vector<MeshLib::Element*> const& elements,
+        AssemblerLib::MeshComponentMap const& mesh_component_map,
+        AssemblerLib::ComponentOrder const order);
+
     template <typename ElementIterator>
```

Comment:
I'd like to move this function definition to the place after the class declaration.